GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -51,9 +51,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// Creates the default options.
 + (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
-                       presentingViewController:(nullable UIViewController *)presentingViewController
+                       presentingViewController:
+                           (nullable UIViewController *)presentingViewController
                                       loginHint:(nullable NSString *)loginHint
-                                       callback:(GIDSignInCallback)callback;
+                                       callback:(nullable GIDSignInCallback)callback;
```

Comment:
You should be able to remove this change now.

Reply:
Done.